lib/devcycle-ruby-server-sdk/localbucketing/platform_data.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+require 'socket'
+require 'json'
+
+module DevCycle
+  class PlatformData
+    attr_accessor :deviceModel, :platformVersion, :sdkVersion, :sdkType, :platform, :hostname
+
+    def initialize(sdk_type, sdk_version, platform_version, device_model, platform, hostname)
+      @sdkType = sdk_type
+      @sdkVersion = sdk_version
+      @platformVersion = platform_version
+      @deviceModel = device_model
+      @platform = platform
+      @hostname = hostname
+    end
+    def default
+      @sdkType='server'
+      @sdkVersion = '1.0.0'
```

Comment:
this should be pulled from the gemspec

Reply:
was place holder - will swap for VERSION